Tests/BodegaTests/DiskStorageTests.swift

```diff
@@ -112,6 +112,67 @@ final class DiskStorageTests: XCTestCase {
         XCTAssert(allKeys[3].value == "3")
         XCTAssert(allKeys.count == 10)
     }
+    
+    func testModificationDate() async throws {
+        // Make sure the modificationDate is nil if the key hasn't been stored
+        var mod = await storage.modificationDate(key: Self.testCacheKey)
+        XCTAssertNil(mod)
+        
+        // Make sure the modification date is in the right range if it has been stored
+        var pre = Date()
+        try await storage.write(Self.testData, key: Self.testCacheKey)
+        var post = Date()
+        mod = await storage.modificationDate(key: Self.testCacheKey)
+        XCTAssertNotNil(mod)
+        XCTAssertLessThanOrEqual(pre, mod!)
+        XCTAssertLessThanOrEqual(mod!, post)
+        
+        try await Task.sleep(nanoseconds: 1_000_000_000)
+        
+        // Make sure the modification date is updated when the data is re-written
+        pre = Date()
+        try await storage.write(Self.testData, key: Self.testCacheKey)
+        post = Date()
+        mod = await storage.modificationDate(key: Self.testCacheKey)
+        XCTAssertNotNil(mod)
+        XCTAssertLessThanOrEqual(pre, mod!)
+        XCTAssertLessThanOrEqual(mod!, post)
+    }
+    
+    func testAccessDate() async throws {
+        // Make sure the accessDate is nil if the key hasn't been stored
+        var acc = await storage.accessDate(key: Self.testCacheKey)
```

Comment:
Same comment here as I mentioned for `mod` above.

Reply:
Done